pkg/internal/common/logging/modlog/deflog.go

```diff
@@ -0,0 +1,145 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package modlog
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/hyperledger/aries-framework-go/pkg/internal/common/logging/metadata"
+
+	logapi "github.com/hyperledger/aries-framework-go/pkg/common/log"
+)
+
+const (
+	logLevelFormatter   = "UTC %s-> %s "
+	logPrefixFormatter  = " [%s] "
+	callerInfoFormatter = "- %s "
+)
+
+//defLog is a standard default logger implementation
```

Comment:
It would be helpful to comment further on:
- what this package is doing (It appears you are wrapping the std library logger to support ...).
- a "default logger implementation" means. (it's a bit of a funny name for a wrapper package -- in an isolated PR anyways).
- why modLog wasn't leveraged (as there appears to be overlap) - performance?

Reply:
Will fix comments.
Using different separate logger implementation for default and custom to avoid using locks and performance of course.